core/sequencer/trillian_client.go

```diff
@@ -28,6 +28,27 @@ import (
 	tclient "github.com/google/trillian/client"
 )
 
+// TrillianFactory creates verifying clients for Trillian
+type TrillianFactory interface {
+	MapClient(ctx context.Context, dirID string) (TrillianMap, error)
+	LogClient(ctx context.Context, dirID string) (TrillianLog, error)
+}
+
+// TrillianMap communicates with the Trilian map and verifies the responses.
+type TrillianMap interface {
```

Comment:
I'm trying to abstract away the *verifying client* so that I can mock out the calls to the Trillian Log and Map servers without having to cryptographically sign the fake responses and consistency proofs.

Reply:
strictly speaking they do not

Reply:
I think that lower-casing them could help avoid some confusion when facing the variety of Trillian "client" types.